sources/packages/backend/apps/workers/src/controllers/assessment/assessment.dto.ts

```diff
@@ -160,6 +160,26 @@ export interface WorkflowWrapUpJobInDTO {
   [WORKFLOW_DATA]: WorkflowData;
 }
 
+/**
+ * Types of operations that can be performed when wrapping up a workflow.
+ */
+export enum WorkflowWrapUpType {
+  /**
+   * Executes only a subset of the wrap-up operations, setting the assessment status to completed.
+   * Used when the workflow does not reach the calculations stage and must be aborted.
+   */
+  AssessmentStatusOnly = "Assessment status only",
+  /**
+   * Executes all wrap-up operations, setting the assessment status to completed and
+   * executing other processes related to sequential processing.
+   */
+  CompleteWrapUp = "Complete wrap-up",
```

Comment:
Minor, it could be just `Complete` as it belongs to wrap up type.

Reply:
Changed.